performance-test/locustfile.py

```
@@ -45,7 +54,9 @@ class MonewUser(HttpUser):
   host = "http://localhost:8080"
 
   def on_start(self):
-    self.user_id = random.choice(user_id_pool)
+    idx = random.randint(0, len(user_id_pool) - 1)
```

Ruff (0.15.12)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)